web-client/src/firebase/index.ts

```diff
@@ -9,8 +8,8 @@ const config = {
     projectId: 'reach4help-dev',
     storageBucket: 'reach4help-dev.appspot.com',
     messagingSenderId: '664078449280',
-    appId: '1:664078449280:web:fad2c3b30771213e5f4465',
-    measurementId: 'G-WFTRYNEQBB',
+    appId: '1:664078449280:web:115f33485dfc48e35f4465',
+    measurementId: 'G-DKH4PLZ62L',
```

Comment:
I think it isn't a good idea to have these keys and ids publicly in our repo 🤔

Reply:
Maybe this can go into .env files, what do ya think?

Reply:
I talked with Joseph and he'll handle this 😄 . Also, I don't if it is a good idea to maintain these values in the future. I think we should refresh the app id's and keys after they get removed from the code base. Maybe @mdeous can support me here.

Reply:
https://firebase.google.com/docs/hosting/reserved-urls

Reply:
> I talked with Joseph and he'll handle this 😄 . Also, I don't if it is a good idea to maintain these values in the future. I think we should refresh the app id's and keys after they get removed from the code base. Maybe @mdeous can support me here.

Will @ashwinkjoseph be taking over the work directly in this PR?

Reply:
I think he'll be creating other PR.